h/indexer.py

```diff
@@ -32,15 +31,11 @@ def delete_annotation(id_):
 
 
 @celery.task
-def reindex():
+def reindex_annotations():
     if not celery.request.feature('postgres'):
```

Comment:
I still want to keep the celery task for reindex to check for the feature. The CLI command on the other hand does run regardless of the flag.